README.md

```diff
@@ -82,6 +82,14 @@ This plugin will look for linters in the [`ale_linters`](ale_linters) directory.
 Each directory within corresponds to a particular filetype in Vim, and each file
 in each directory corresponds to the name of a particular linter.
 
+### Always showing gutter
+To prevent left gutter to disappear when there is no errors set
```

Comment:
Fixed readme. Options section would probably be useful. On the note of keeping track if all errors are gone perhaps we should look into adding some info into status line. Could be nice to have it as option.